package.json

```diff
@@ -12,7 +12,8 @@
     "coverage": "jest --coverage",
     "prettier": "prettier --check .",
     "prettier:fix": "prettier --write .",
-    "pre-commit": "npm run prettier:fix && npm run lint && npm run test"
+    "lint-staged": "npx lint-staged",
+    "pre-commit": "npm run lint-staged && npm run test"
```

Comment:
Where is prettier? Or you put it somewhere else?

Reply:
Next suggests running next lint in a defined buildEslintCommand in .lintstagedrc.js; I also put prettier command in that document.